MetroscopeModelingLibrary/MultiFluid/HeatExchangers/Fogging.mo

```diff
@@ -0,0 +1,363 @@
+within MetroscopeModelingLibrary.MultiFluid.HeatExchangers;
+model Fogging
+  package FlueGasesMedium =
+      MetroscopeModelingLibrary.Utilities.Media.FlueGasesMedium;
+  package WaterSteamMedium =
+      MetroscopeModelingLibrary.Utilities.Media.WaterSteamMedium;
+  import MetroscopeModelingLibrary.Utilities.Units;
+  import MetroscopeModelingLibrary.Utilities.Units.Inputs;
+
+  // Boundary Conditions
+  Inputs.InputMassFlowRate Q_fg_in(start=Q_fg_in_0) "Inlet fg mass flow rate at the inlet";
+
+  // Parameters
+  Inputs.InputMassFraction x_vapor(start=1); // Vapor mass fraction
+
+  // Definitions
+  Units.SpecificEnthalpy h_vap_sat;
+  Units.SpecificEnthalpy h_liq_sat;
+  Units.MassFlowRate Q_w_in(start=Q_w_in_0) "Inlet water mass flow rate at the inlet";
+
+  // Initialization parameters
+  // Flow Rates
+  parameter Units.MassFlowRate Q_fg_in_0 = 500;
+  parameter Units.MassFlowRate Q_w_in_0 = 1;
+
+
+  WaterSteam.Connectors.Inlet C_water_in annotation (Placement(transformation(extent={{-10,50},{10,70}}), iconTransformation(extent={{-10,50},{10,70}})));
+  WaterSteam.Pipes.HeatLoss water_evaporation annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=270,
+        origin={0,-14})));
+  WaterSteam.BoundaryConditions.Sink sink_w annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=270,
+        origin={0,-44})));
+  FlueGases.Connectors.Inlet C_fg_inlet annotation (Placement(transformation(extent={{-110,-10},{-90,10}}), iconTransformation(extent={{-110,-10},{-90,10}})));
+  FlueGases.Pipes.HeatLoss evaporative_cooling annotation (Placement(transformation(extent={{-60,-10},{-40,10}})));
+  FlueGases.BoundaryConditions.Source source_fg annotation (Placement(transformation(extent={{-10,-10},{10,10}},
+        rotation=270,
+        origin={40,44})));
+  FlueGases.Connectors.Outlet C_fg_out annotation (Placement(transformation(extent={{90,-10},{110,10}}), iconTransformation(extent={{90,-10},{110,10}})));
+  WaterSteam.Pipes.PressureCut fogging_nozzle_w annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=270,
+        origin={0,14})));
+  FlueGases.Pipes.PressureCut pressureCut annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=270,
+        origin={40,20})));
```

Comment:
Add `Q_w_in_0` and `Q_fg_in_0` modifiers in components definition to fully benefit from them